pdsfile/pds4file/rules/cassini_iss_fring_mosaics_rsfrench2025.py

```
@@ -0,0 +1,359 @@
+##########################################################################################
+# Update the global dictionary of subclasses
+##########################################################################################
+
+pds4file.Pds4File.SUBCLASSES['cassini_iss_fring_mosaics_rsfrench2025'] = cassini_iss_fring_mosaics_rsfrench2025
+
+##########################################################################################
+# Unit tests
+##########################################################################################
+
+import pytest
+from .pytest_support import *
```

Flake8 (7.3.0)

[error] 'from .pytest_support import *' used; unable to detect undefined names

(F403)